hack/common

```diff
@@ -42,7 +42,8 @@ k8s_nodename() {
 
 k8s_env() {
     node_name=$(k8s_nodename)
-    if [[ "$node_name" == "$MINIKUBE" ]]; then
+    # for minikube >= v1.8.0 the default node name is "m01", for minikube < v1.8.0 - "minikube"
+    if [[ "$node_name" == "$MINIKUBE" || "$node_name" == "m01" ]]; then
```

Comment:
@ialidzhikov I am using the dev-setup with a local k8s cluster with minikube on Docker. I observed that I had to change the node name back to minikube(v1.26.1), for the `make dev-setup` to run. As per minikube [docs](https://minikube.sigs.k8s.io/docs/faq/#how-can-i-create-more-than-one-cluster-with-minikube), the default node is minikube. Shall i create a PR for this?

Reply:
This check still supports Node name "minikube".

https://github.com/gardener/gardener/blob/e2017297ebab8502b0078ee39ba1e7e184462857/hack/local-development/common/helpers#L47-L51